config_test.go

```diff
@@ -0,0 +1,131 @@
+package frontman
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/troian/toml"
+)
+
+func TestNewMinimumConfig(t *testing.T) {
+	envURL := "http://foo.bar"
+	envUser := "foo"
+	envPass := "bar"
+
+	// TODO: Not sure if this is really a good idea... could mess with other things
+	os.Setenv("FRONTMAN_HUB_URL", envURL)
+	os.Setenv("FRONTMAN_HUB_USER", envUser)
+	os.Setenv("FRONTMAN_HUB_PASSWORD", envPass)
+
+	mvc := NewMinimumConfig()
+
+	assert.Equal(t, envURL, mvc.HubURL, "HubURL should be set from env")
+	assert.Equal(t, envUser, mvc.HubUser, "HubUser should be set from env")
+	assert.Equal(t, envPass, mvc.HubPassword, "HubPassword should be set from env")
+
+	// Unset in the end for cleanup
+	defer os.Clearenv()
```

Comment:
I was thinking about this too.
I oriented myself on: https://github.com/kelseyhightower/envconfig/blob/master/envconfig_test.go#L85

Guess the assumption is that the tests are run on CI. But I understand that this can cause problems when you run the tests on your local machine 🤔

Reply:
ok, so `os.Clearenv()` doesn't clear the env variables in the parent process. Only in the current and child ones.
Just tested on my local computer. And it doesn't remove anything.

So the only problem could be is that it could break the next tests if they depended on some env vars (but this is a bad practice BTW)

Reply:
yeap